AC3.2Unit2FinalAssessment/AC3.2Unit2FinalAssessment/ColorTableViewController.swift

```diff
@@ -0,0 +1,104 @@
+        return crayons.count
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "colorCell", for: indexPath)
+        
+        let crayonColor: Crayon = crayons[indexPath.row]
+        let red = CGFloat(crayonColor.red)
+        let green = CGFloat(crayonColor.green)
+        let blue = CGFloat(crayonColor.blue)
+        cell.backgroundColor = UIColor(red: red, green: green, blue: blue, alpha: 1.0)
+        cell.textLabel?.text = crayonColor.name
+        return cell
+    }
+    
+    /*
+     // Override to support conditional editing of the table view.
+     override func tableView(_ tableView: UITableView, canEditRowAt indexPath: IndexPath) -> Bool {
+     // Return false if you do not want the specified item to be editable.
+     return true
+     }
+     */
+    
+    /*
+     // Override to support editing the table view.
+     override func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCellEditingStyle, forRowAt indexPath: IndexPath) {
+     if editingStyle == .delete {
+     // Delete the row from the data source
+     tableView.deleteRows(at: [indexPath], with: .fade)
+     } else if editingStyle == .insert {
+     // Create a new instance of the appropriate class, insert it into the array, and add a new row to the table view
+     }
+     }
+     */
+    
+    /*
+     // Override to support rearranging the table view.
+     override func tableView(_ tableView: UITableView, moveRowAt fromIndexPath: IndexPath, to: IndexPath) {
+     
+     }
+     */
+    
+    /*
+     // Override to support conditional rearranging of the table view.
+     override func tableView(_ tableView: UITableView, canMoveRowAt indexPath: IndexPath) -> Bool {
+     // Return false if you do not want the item to be re-orderable.
+     return true
+     }
+     */
+    
+    
+    // MARK: - Navigation
+    
+    // In a storyboard-based application, you will often want to do a little preparation before navigation
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if let tappedColorCell = sender as? UITableViewCell {
+            if segue.identifier == "colorShowSegue" {
+                let destinationVC = segue.destination as! CrayonViewController
+                let cellIndexPath = self.tableView.indexPath(for: tappedColorCell)!
+                let selectedColor: Crayon = crayons[cellIndexPath.row]
+                destinationVC.labelContent = selectedColor.name
+                
+                let crayonColor: Crayon = crayons[cellIndexPath.row]
+                let red = CGFloat(crayonColor.red)
+                let green = CGFloat(crayonColor.green)
+                let blue = CGFloat(crayonColor.blue)
+                destinationVC.view.backgroundColor = UIColor(red: red, green: green, blue: blue, alpha: 1.0)
```

Comment:
It is better style to just pass the entire crayon object to the next vc and then set backgroundColor and labelContent in viewDidLoad: